packages/vite/schemas/manifest.schema.json

```diff
@@ -0,0 +1,65 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://vitejs.dev/schemas/v2.0.0/manifest.json",
```

Comment:
Ideally, it should be possible to retrieve the JSON schemas at this URL.

I've versioned the URL to allow for future schemas under a similar URL but I am unsure about how we want to version the schemas.

### Mirroring Vite versions

This would probably add some maintenance burden if not automated. Not sure this is a good option because the schema would not necessarily change with each new Vite version.

### Sync only the major version

This option would consist in keeping only the major version in sync with Vite, and minor and path version would be incremented when actual modifications are made to the schemas.

For example, schema `v2.0.0` would currently be valid for Vite `v2.9.6` generated manifests.


Let's hear pros and cons.